.drone.yml

```diff
@@ -100,6 +100,19 @@ pipeline:
     files:
       - 'h5c/vic/src/vic-webapp/coverage/lcov.info'
 
+  trigger-downstream:
+    image: 'gcr.io/eminent-nation-87317/vic-downstream-trigger:1.1'
+    environment:
+      SHELL: /bin/bash
+      DOWNSTREAM_REPO: vmware/vic
+    secrets:
+      - drone_server
+      - drone_token
+    when:
+      repo: vmware/vic-ui
+      branch: [master, 'releases/*']
```

Comment:
Sorry @zjs it should be a better idea to trigger the downstream only with "Push" event, guaranteeing in this way only to trigger all of these processes when the changes will merge to master in another way we will be triggering it all the time, also for simple commits.

Reply:
Definitely! I'll remove the "TEMP: force downstream trigger" commit at the end of the review process (since I need it for now to test again if anyone requests changes).